### PR TITLE
the cooler Balance tweaks (WIP)

### PR DESCRIPTION
**Pull request by TheUnholyWrath**
just kinda aimlessly nerfing the things I like right now

- revert slardar bash cd buff
- nerf bane ult damage at oaa levels
- nerf mirana fade time and duration on ulti
- tiny rescale on slardar ult with talent
- nerf invis duration on silver edge 
- small nerfs to small cleaver
- nerf razor link at max level
- move blood sword up to tier 4, add an awful item to replace it (way too many good right click items at t3)

### CODE REVIEW

**Comment by lovesmuffins** [comment]
I feel like slardar bash is in a weird place for balance and it will never truly be good, that or it'll be completely OP. So I Did some brainstorming and came up with a solution.

5 sec CD
Bash Duration is 1 second flat
Every attack reduces the CD by 0.1/0.2/0.3/0.4/0.5/0.6
This change, even at max attack speed, won't let you perma stun but will still let you bash more often. This is better than just buffing the CD since it can be altered by disarm, attack speed slows, etc., and also to truly maximize the ability potential you'd have to go full attack speed which is not worth it.

I also theorized switching out a talent for a +0.2 stun duration but that's just for flavor and isn't important to my suggestion.

File: game/scripts/npc/abilities/slardar_bash_oaa.txt [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "4.0"
+    "AbilityCooldown"                                     "4.4"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/slardar_bash_oaa.txt:25]: lmao, 4 or 5 seconds is fine, putting decimals is weird

Comment (lovesmuffins) [reply game/scripts/npc/abilities/slardar_bash_oaa.txt:25]: wow ignorin me

**Comment by DarkoniusXNG** [comment]
> I feel like slardar bash is in a weird place for balance and it will never truly be good, that or it'll be completely OP. So I Did some brainstorming and came up with a solution.
> 
> 4 sec CD Bash Duration is 1 second flat Every attack reduces the CD by 0.1/0.2/0.3/0.4/0.5/0.6 This change, even at max attack speed, won't let you perma stun but will still let you bash more often. This is better than just buffing the CD since it can be altered by disarm, attack speed slows, etc., and also to truly maximize the ability potential you'd have to go full attack speed which is not worth it.
> 
> I also theorized switching out a talent for a +0.2 stun duration but that's just for flavor and isn't important to my suggestion.

I like current bash implementation because it's not op or annoying and it can still be good if you are good. Your suggestion, beside adding unnecessary complexity, will make Bash more annoying like vanilla Bash. In which universe going full attack speed on vanilla slardar is not worth it? Especially in a game that goes past lvl 30 and where items can be upgraded? I don't have more good arguments why I don't like your suggestion. I don't want to be permabashed in any way, doesn't matter how many 'ifs' the ability has.

File: game/scripts/npc/items/item_satanic.txt [hunk 2]
```diff
@@ -57,7 +57,7 @@
     "AbilityValues"
     {
       "bonus_strength"                                    "25 35 50 70 95"
-      "bonus_damage"                                      "20 40 70 110 160" //OAA
+      "bonus_damage"                                      "20 36 60 92 132" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/items/item_satanic.txt:60]: Revert this

File: game/scripts/npc/neutral_items.txt [hunk 3]
```diff
@@ -170,14 +175,15 @@
...
-      //"item_ballista"                  "1"
       "item_reflex_core"                                  "1" // replaces item_demonicon
       "item_fallen_sky"                                   "1"
       "item_force_field"                                  "1"
-      //"item_pirate_hat"                "1"
       "item_ex_machina"                                   "1"
       "item_giants_ring"                                  "1"
       "item_book_of_shadows"                              "1"
+
+      //"item_ascetic_cap"               "1"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/neutral_items.txt:185]: You probably wanted to write ``item_ballista`` here

File: game/scripts/npc/items/neutral/item_blood_sword.txt [hunk 4]
```diff
@@ -36,7 +36,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "50"
+        "bonus_damage"                                    "40"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/items/neutral/item_blood_sword.txt:39]: You are moving Blood Sword to tier 4, why nerf the dmg too?

**Review by DarkoniusXNG — CHANGES_REQUESTED** [review]

**Pull request merged**